drake/systems/plants/KinematicsCache.h

```diff
@@ -53,6 +53,9 @@ class KinematicsCacheElement
   {
     // empty
   }
+
+public:
+	EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
was this actually necessary? Do you know at this point if not having this can cause trouble at runtime? Just worried about my assumption that the compiler will always tell you messed up.

Also, indentation.